### PR TITLE
[GEOS-11101] metadata: multiple tabs

### PR DESCRIPTION
**Pull request by NielsCharlier**
[GEOS-11101](https://osgeo-org.atlassian.net/browse/GEOS-11101) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=geoserver&utm_campaign=badge_info)

  

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [x] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [x] New unit tests have been added covering the changes.
- [x] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [x] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [x] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [x] Bug fixes and small new features are presented as a single commit.
- [x] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Comment by aaime** [comment]
As a supported module, new functionality should come with documentation updates.

**Comment by NielsCharlier** [comment]
is this okay to merge?

File: doc/en/user/source/extensions/metadata/uiconfiguration.rst [hunk 1]
```diff
@@ -115,6 +116,11 @@ For example:
 
   condition: equalTo(typeOf("."), 'FeatureTypeInfo')
 
+tab
+^^^
+Optionally, attributes may be displayed on separate tabs. All tabs must be listed under `tabs` in the main configuration. Then this
+property is used to assign each attribute to a tab.
+
```
Comment (aaime) [inline doc/en/user/source/extensions/metadata/uiconfiguration.rst:123]: A screenshot showing how multiple tabs are rendered in the UI would not hurt

File: src/extension/metadata/src/test/java/org/geoserver/metadata/AbstractMetadataTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2018 Open Source Geospatial Foundat-ion - all rights reserved
```
Comment (aaime) [inline src/extension/metadata/src/test/java/org/geoserver/metadata/AbstractMetadataTest.java:1]: Nope

**Review by aaime — CHANGES_REQUESTED** [review]

**Pull request merged**